src/contract_wrappers/token_wrapper.ts

```diff
@@ -80,6 +81,20 @@ export class TokenWrapper extends ContractWrapper {
         });
     }
     /**
+     * Sets the spender's allowance to an unlimited number of baseUnits on behalf of the owner address.
+     * Equivalent to the ERC20 spec method `approve`.
```

Comment:
I would still like us to add a short line along these lines:

`Setting an unlimited allowance will lower the gas cost for filling orders involving tokens that forego updating allowances set to the max amount (e.g ZRX, WETH)`

Otherwise it is not clear why we expose this method.